README.md

```diff
@@ -40,8 +40,8 @@ How does the package use scverse data structures (please describe in a few sente
 -   [ ] The code is publicly available under an [OSI-approved](https://opensource.org/licenses/alphabetical) license
 -   [ ] The package provides versioned releases
 -   [ ] The package can be installed from a standard registry (e.g. PyPI, conda-forge, bioconda)
--   [ ] The package uses automated software tests and runs them via continuous integration (CI)
--   [ ] The package provides API documentation via a website or README
+-   [ ] The package uses automated software tests that cover all public functions and runs them via continuous integration (CI)[^1]
```

Comment:
It is the correct expectation but a rather strong one. I'm fine with it but wonder what other people think. Maybe a quick question for zulip?